dash/frontend/src/app/modules/private/pages/api-key-management/api-key-form/api-key-form.component.html

```diff
@@ -1,39 +1,40 @@
-<div class="row h-xs-100 display-xs-block overflow-xs-auto me-xs-1">
-  <div class="col-xs ms-xs-6 mt-xs-6">
-    <div class="m-xs-0 overflow-x-xs-auto">
-      <div >
+<div class="row mt-xs-7 mx-xs-1 h-xs-100 display-xs-block overflow-xs-auto">
+  <div class="col-xs mb-xs-4">
+    <div class="row">
+      <div class="col-xs">
```

Comment:
This is an example of the div wrapping I'm talking about. Why are there 4 layers of divs around the mat-card?

Reply:
@becky-intelletive Got it.